homeassistant/components/nzbget/const.py

```diff
@@ -14,7 +14,7 @@
 DEFAULT_SCAN_INTERVAL = 5  # time in seconds
 DEFAULT_SPEED_LIMIT = 1000  # 1 Megabyte/Sec
 DEFAULT_SSL = False
-DEFAULT_VERIFY_SSL = False
+DEFAULT_VERIFY_SSL = True
```

Comment:
How will this impact existing setups?  Will this cause existing setups to fail if they do not have good HTTPS certificates?

Reply:
this is only used in the config_flow when setting up a connection. The default is set to true to accommodate common situations like reverse proxying. This is irrelevant when not using SSL as there is no validation of certificates when only using http connections.